src/index.ts

```diff
@@ -17,8 +17,11 @@ import { TSLIB, TSLIB_VIRTUAL, tslibSource, tslibVersion } from "./tslib";
 import { createFilter } from "./get-options-overrides";
 
 // these use globals during testing and are substituted by rollup-plugin-re during builds
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
maybe we should just disable this rule entirely. an explicit `any` is _explicit_ after all